src/main/java/teammates/common/util/Utils.java

```diff
@@ -35,5 +37,23 @@ public static Gson getTeammatesGson() {
                 .disableHtmlEscaping()
                 .create();
     }
+    
+    /**
+     * Convert a list of {@link FeedbackParticipantType} to a string where each element will be separated by
+     * a delimiter.
+     */
+    public static String joinParticipantTypes(List<FeedbackParticipantType> participants, String delimiter) {
+        String result = "";
+        if (!participants.isEmpty()) {
```

Comment:
I see, but my point of view is having a one exit route out of a function is better for readability